x-pack/plugins/apm/public/components/app/Main/LicenseCheck/index.tsx

```diff
@@ -6,18 +6,19 @@
 import React from 'react';
 import { STATUS } from '../../../../constants/index';
 import { LicenceRequest } from '../../../../store/reactReduxRequest/license';
+import { InvalidLicenseNotification } from './InvalidLicenseNotification';
 
-function LicenseChecker() {
+export const LicenseCheck: React.SFC = ({ children }) => {
```

Comment:
Wanna check and see if `React.FunctionalComponent` (I think that's what the new one is) is available in our current types? SFC is being deprecated/removed now b/c of hooks, AFAIK.

Reply:
Hmm 🤔 

![screenshot 2019-02-26 at 14 56 46](https://user-images.githubusercontent.com/209966/53417883-ce64ae00-39d6-11e9-8cb3-910216fe0ff1.png)

Reply:
Oh, `React.FunctionComponent` works. Is that the same?

Reply:
`React.FunctionComponent` is just an alias for `React.SFC`, right?

Reply:
Changed to `React.FunctionComponent` btw.

Reply:
Might just be an alias but I think because they are no longer "stateless" by definition, the TS team is deprecating SFC (that's how I understand it) :+1: